CMSgov/onemac
# Oy2 1004 common alerts

Changes:
1. Replaced onError with the new singleton AlertBar.
2. Some files are going away with the OKTA change: profile, signup
3. Cleaned up some dead files.
4. Removed unused urgent field for SPAs.
5. Dismiss the alert on page change.
6. Fixed Dashboard links which need to be a history push instead of normal links.  This also fixes the back button issue where a blank page shows up. 

Notes: The alert bar can only be included once in an application.  Code then just calls the static alert or dismiss methods with the proper data.

Tests:
Testing is not easy as some of the errors will be hard to trigger in the AWS environment.  This test is to show at least the alert once.  For seeing the errors it is easier to change the code directly to force the alert to show for errors like submission or fetch errors.
1. Login to the SPA form.
2. Submit a new entry (any will do).
3. Verify a green success alert is shown below the header.
4. Navigate to the about page and verify the alert is no longer visible.

Test endpoint: https://d2awwy0flo8ssy.cloudfront.net
